### ISSUE TITLE
SceneViewer has glitchy opaque/transparent sorting order

### ISSUE THREAD

**Issue by soraryu** [google/model-viewer#1066]
### Description

Related to #1065 but for SceneViewer.
The model has issues with the ground plane shadow, and additionally a lot of sorting order / transparency glitches (much more broken in SceneViewer than model-viewer and definitely much much more than directly in SketchFab).

SceneViewer (same happens in AR):
![Screenshot_2020-03-01-09-43-09-837_com google android googlequicksearchbox](https://user-images.githubusercontent.com/2693840/75622494-36794180-5ba1-11ea-8050-073ded1900f1.jpg)

Sketchfab "Ground Truth":
![IMG_20200301_093605](https://user-images.githubusercontent.com/2693840/75622502-4abd3e80-5ba1-11ea-84e1-bb194f18fdb3.jpg)

Note: we actually found this issue while on a client project with a different model, but the model below is similar in broken behavior.

#### Live Demo
https://glitch.com/edit/#!/wax-citrine-cress

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [X] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions
latest

**Comment by cdata** [comment]
This looks like it might just be z-fighting with the shadow plane. WDYT @elalish ?

**Comment by soraryu** [comment]
Yeah, the other one I opened is for model-viewer (where similar stuff happens plus the floor object is totally missing): https://github.com/google/model-viewer/issues/1065
